packages/offix-datastore/src/replication/GraphQLCrudQueryBuilder.ts

```diff
@@ -43,7 +43,9 @@ export class GraphQLCrudQueryBuilder implements GraphQLQueryBuilder {
                 }`
       };
 
-      const queries = {
+      const queries: ReplicatorQueries = {
+        // TODO create a GraphQLCrudQuery class implementing ReplicatorQuery
```

Comment:
+1

FYI - GraphQL crud will deal with the mapping for different providers in near future but having offix working with any user-provided server would be amazing +100 feature